include/mpsl/mpsl_cx_config_bluetooth.h

```diff
@@ -30,6 +30,18 @@
 
 int32_t mpsl_cx_bt_interface_3wire_config_set(void);
 
+/** @brief Configures the Bluetooth Radio One Wire Coexistence interface.
+ *
+ * This function sets device interface parameters for the Coexistence module.
+ * The module is used to control PTA interface through the given pins and resources.
+ *
+ * @retval   0             Coexistence interface successfully configured.
+ * @retval   -NRF_EPERM    Coexistence interface is not available.
+ *
+ */
+
+int32_t mpsl_cx_bt_interface_1wire_config_set(void);
```

Comment:
I have a slight preference to move `mpsl_cx_bt_interface_1wire_config_set` above the 3-wire interface as it is simpler.

Reply:
will update also in other places to have it aligned